source/index.html

```diff
@@ -163,41 +150,39 @@ <h2 class="map__title">приглашаем к сотрудничеству ди
 
     <footer class="page-footer">
       <div class="page-footer__wrapper">
-        <div class="page-footer__logo">
+        <a class="page-footer__logo-link" href="#">
           <span class="visually-hidden">Логотип Кэт Энерджи</span>
-          <a class="page-footer__logo-link" href="#">
-            <svg class="page-footer__logo-image" aria-hidden="true" focusable="false" width="128" height="24"></svg>
-          </a>
-        </div>
+          <svg class="page-footer__logo-image" aria-hidden="true" focusable="false" width="128" height="24"></svg>
+        </a>
 
         <div class="social">
           <ul class="social__list">
             <li class="social__item">
-              <span class="visually-hidden">Мы в Вконтакте</span>
-              <a class="social__link social__link--vkontakte" href="">
+              <a class="social__link social__link--vkontakte" href="#">
+                <span class="visually-hidden">Мы в Вконтакте</span>
                 <svg class="social__link-icon">
                 </svg>
               </a>
             </li>
             <li class="social__item">
-              <span class="visually-hidden">Мы в Инстаграме</span>
-              <a class="social__link social__link--instagram" href="">
+              <a class="social__link social__link--instagram" href="#">
+                <span class="visually-hidden">Мы в Инстаграме</span>
                 <svg class="social__link-icon">
                 </svg>
               </a>
             </li>
             <li class="social__item">
-              <span class="visually-hidden">Мы в Фейсбуке</span>
-              <a class="social__link social__link--facebook" href="">
+              <a class="social__link social__link--facebook" href="#">
+                <span class="visually-hidden">Мы в Фейсбуке</span>
                 <svg class="social__link-icon"></svg>
               </a>
             </li>
           </ul>
         </div>
         <div class="copyright">
-          <a class="copyright__link" href="">
+          <a class="copyright__link" href="#">
             <p class="copyright__text">HTML Academy</p>
-            <img class="copyright__link-developer" src=""></img>
+            <img class="copyright__link-developer" src="#" alt="Логотип HTML Academy"></img>
```

Comment:
Погоди, а с каких пор у `<img>` появилась закрывающая половина тега?
```suggestion
            <img class="copyright__link-developer" src="#" alt="Логотип HTML Academy">
```

Reply:
Ой